Directory.Build.props

```diff
@@ -2,10 +2,9 @@
   <Import Project="eng\Versions.props" />
 
   <PropertyGroup>
-
     <BuildForWinUI Condition="'$(SolutionFileName)' == 'Microsoft.Maui.WinUI.sln' AND '$(Packing)' == ''">true</BuildForWinUI>
     <BuildForAndroid Condition="'$(SolutionFileName)' == 'Microsoft.Maui.Droid.sln' AND '$(Packing)' == ''">true</BuildForAndroid>
-    <BuildForNet6 Condition="'$(Packing)' == 'true' OR $(SolutionFileName.Contains('net6')) == true OR '$(BuildForWinUI)' == 'true' OR '$(BuildForAndroid)' == 'true'">true</BuildForNet6>
+    <BuildForNet6 Condition="'$(Packing)' == 'true' OR $(MSBuildProjectFile.Contains('SingleProject')) == true  OR $(MSBuildProjectFile.Contains('net6')) == true OR $(SolutionFileName.Contains('net6')) == true OR '$(BuildForWinUI)' == 'true' OR '$(BuildForAndroid)' == 'true'">true</BuildForNet6>
```

Comment:
This `Condition` is starting to get out of hand, can we invert the condition here so it will be a lot shorter? Check for the legacy Xamarin `.sln` or something?

Reply:
Probably, but since it's a weird hack either way I'd rather not mess with it. I'll probably do it wrong a few times while trying to invert it :-)  Ideally we can just delete this whole thing in a few weeks and all of our legacy slns?!?